### PR TITLE
Bump to kubernetes 1.16.0 rc.2

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by deads2k** [comment]
we will need a bug for the CRD issue.  I think we more the generation to openshift/api on Monday.

**Comment by deads2k** [comment]
@sttts the openapi bits are going to make you cranky, but landing this before we open up is more important than describe.

**Comment by deads2k** [comment]
Based on the last push time, we're short several patches.  On Monday, please be sure to push them.

https://github.com/openshift/origin/commit/1f24b290b703f80d5fa5ae71adcad3a8cda8128f is the last one I know of so far.

**Comment by p0lyn0mial** [comment]
> Based on the last push time, we're short several patches. On Monday, please be sure to push them.

Sure, np, I'm going to add those directly into this pull since adding into the fork might not work because of the automation (publishing bots).

File: glide.yaml [hunk 1]
```diff
@@ -8,92 +8,98 @@ import:
...
 # openshift second
 - package: github.com/openshift/api
-  version: master
+  version: prebase-1.16
 - package: github.com/openshift/client-go
-  version: master
+  version: prebase-1.16.0-beta.2
 - package: github.com/openshift/library-go
-  version: master
+  version: prebase-1.16.0-beta.2
```
Comment (mfojtik) [inline glide.yaml]: use `prebase-1.16`

File: glide.yaml [hunk 2]
```diff
@@ -8,92 +8,98 @@ import:
...
 - package: github.com/openshift/apiserver-library-go
-  version: master
+  version: prebase-1.16.0-beta.2
```
Comment (mfojtik) [inline glide.yaml]: use `prebase-1.16`

**Comment by mfojtik** [comment]
/retest

AWS flake

**Comment by mfojtik** [comment]
the image-ecosystem seems to be failing on: ```
Cloning "https://github.com/sclorg/cakephp-ex.git" ...
error: fatal: unable to access 'https://github.com/sclorg/cakephp-ex.git/': Could not resolve host: github.com; Unknown error
```

**Comment by deads2k** [comment]
Images has worked in the past.  I will merge when we're ready.

**Pull request merged**